package.json

```diff
@@ -11,6 +11,7 @@
         "@mui/icons-material": "^5.8.4",
         "@mui/lab": "^5.0.0-alpha.93",
         "@mui/material": "^5.9.3",
+        "@mui/system": "^5.11.5",
```

Comment:
if this was a new NPM package that was installed please uninstall it.  If you need any help with this please let me know.